pages/request-workshop.md

```diff
@@ -1,9 +0,0 @@
----
-permalink: /request-workshop/
```

Comment:
For us I think it might be useful if we actually had such a page and just send them to a Google form.

Reply:
That's fine, but we should build that form and the redirect page for ourselves.

Reply:
It's doubtful you can run an HPC Carpentry workshop, but you can run a Carpentries workshop and teach an Incubator lesson (hpc-intro) as part of it.

Restored by cda3e446.